src/ReduxToastr.js

```diff
@@ -19,8 +20,8 @@ export class ReduxToastr extends React.Component {
     timeOut: PropTypes.number,
     confirmOptions: PropTypes.object,
     progressBar: PropTypes.bool,
-    transitionIn: PropTypes.string,
-    transitionOut: PropTypes.string,
+    transitionIn: PropTypes.oneOf(TRANSITIONS.in),
+    transitionOut: PropTypes.oneOf(TRANSITIONS.out),
```

Comment:
yes, you are right. Currently there is a problem: if a transitionOut doesn't hide on 100% (wrong spelling of prebuilt animations or a custom animation is not correct) a toastr box will not be closed. So there are 2 ways: 1st restrict for now the list of transitions and later add "extend transition list" feature, or automatically force hide with timeout. I've decided the first case is better. If you have a different point on it, I can remove props checks and add something like:

 ```js
export function onCSSTransitionEnd(node, callback) {
  var timeoutId = setTimeout(function() {
     if(callback && node) {
        var e = { target: node };
        callback(e);
     }
  }, 10000);

  const runOnce = (e) => {
    clearTimeout(timeoutId);
    e.stopPropagation();
    ReactTransitionEvents.removeEndEventListener(node, runOnce);
    callback && callback(e);
  };
  ReactTransitionEvents.addEndEventListener(node, runOnce);
}
```